api/python/cell_census/src/cell_census/get_anndata.py

```diff
@@ -122,20 +50,18 @@ def get_anndata(
 
     Examples
     --------
-    >>> get_anndata(census, "Mus musculus", obs_value_filter={"tissue": "brain"})
+    >>> get_anndata(census, "Mus musculus", value_filter={"obs": "tissue_general in ['brain', 'lung']"})
 
     >>> get_anndata(census, "Homo sapiens", column_names={"obs": ["tissue"]})
 
-    >>> get_anndata(cesus, "Homo sapiens", obs_joinds=[0,88,222])
-
     """
     exp = get_experiment(census, organism)
-    _obs_query = _build_query(obs_joinids, obs_value_filter)
-    _var_query = _build_query(var_joinids, var_value_filter)
+    _obs_value_filter = None if value_filter is None else value_filter.get("obs", None)
+    _var_value_filter = None if value_filter is None else value_filter.get("var", None)
```

Comment:
Nit (but only needed if we keep the `AxisValueFilters` dict param)
```suggestion
    _obs_value_filter = value_filter.get("obs", None) if value_filter is not None else None
    _var_value_filter = value_filter.get("var", None) if value_filter is not None else None
```